docs/modules/release-notes/pages/0.26.adoc

```diff
@@ -12,7 +12,7 @@ This release brings Windows support, improvements to controlling how Pkl talks o
 
 The next release (0.27) is scheduled for October 10th, 2024.
 
-Please send feedback and questions to https://github.com/apple/pkl/discussions[GitHub Discussions], or submit an issue on https://github.com/apple/pkl/issues/new[Github]. +
+Please send feedback and questions to https://github.com/apple/pkl/discussions[GitHub Discussions], or submit an issue on https://github.com/apple/pkl/issues/new[GitHub]. +
```

Comment:
Some projects don't like changing changelogs. I'm happy to drop this or anything else.